packages/server/src/automations/automationUtils.js

```diff
@@ -0,0 +1,71 @@
+const CouchDB = require("../db")
+
+module.exports.cleanMustache = string => {
```

Comment:
NAB: a little JSdoc to describe the logic of this function might be nice for the uninitiated